cmd/entrypoint/webhooks.go

```diff
@@ -0,0 +1,312 @@
+package entrypoint
+
+import (
+	b "bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net/http"
+	"time"
+
+	k8sTypesCoreV1 "k8s.io/api/core/v1"
+	k8sApiExtTypes "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	k8sApiExtClient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
```

Comment:
These should both say `v1`, not `v1beta1`.

Reply:
Also, `API` not `Api` https://github.com/golang/go/wiki/CodeReviewComments#initialisms